src/ngircd/login.c

```diff
@@ -201,9 +201,40 @@ Login_User_PostAuth(CLIENT *Client)
 	} else
 		IRC_SetPenalty(Client, 1);
 
+// do that autojoin stuff
+  Login_Autojoin(Client);
```

Comment:
This function is lacking a prototype, that’s why the clang build on Travis-CI fails:
https://travis-ci.org/github/ngircd/ngircd/jobs/749462351#L581-L587